packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -647,16 +647,14 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
         // Enable flushing only if we have single commit summary and this is not the initial summary for an empty file
         if (".protocol" in summary.tree && context.ackHandle !== undefined) {
-            let retry = 0;
-            for (;;) {
+            for (let retry = 1; retry <= 4; retry++) {
```

Comment:
this change results in two places mentioning "4" - someone can make a mistake later on and change only one of these values. I'd suggest having infinite loop here.